invenio_app_ils/circulation/api.py

```diff
@@ -62,6 +65,15 @@ def create_loan(params):
         document_pid = circulation_document_retriever(params["item_pid"])
         if document_pid:
             params["document_pid"] = document_pid
+
+    if should_force_checkout:
```

Comment:
actually we could totally get rid off this check and parameter passed to checkout from ui. The next lines decide about that anyway, and I would like to trust more backend than frontend params. @ntarocco WDYT?

Reply:
@vladbmihaescu ignore this one, we just discussed with @ntarocco